components/Possibilities/Possibilities.tsx

```diff
@@ -0,0 +1,122 @@
+/* eslint-disable jsx-a11y/media-has-caption */
+import Image from 'next/image';
+import BarcodeImageSrc from '../../public/images/possibilities-barcode.jpg';
+import ProductCardWhiteImageSrc from '../../public/images/possibilities-product-card-white.jpg';
+import ProductCardExpiresSrc from '../../public/images/possibilities-status-expires.jpg';
+import ProductCardExpiredSrc from '../../public/images/possibilities-status-expired.jpg';
+import TieIconSrc from '../../public/images/possibilities-tie.svg';
+import CapIconSrc from '../../public/images/possibilities-cap.svg';
+import InterfaceImageSrc from '../../public/images/possibilities-interface.jpg';
+
+export function Possibilities() {
+  return (
+    <section className="possibilities">
+      <div className="container possibilities__wrapper">
+        <h2 className="font-family-2 text-type-2 possibilities__title">Best By отслеживает срок годности продукции за вас</h2>
+        <div className="font-family-1 possibilities__cards">
+          <div className="possibilities__card">
+            <span className="text-type-3 possibilities__label possibilities__label--white">Быстрое добавление товара</span>
+            <p className="text-type-6 possibilities__description possibilities__description--white">
```

Comment:
You can use the modifier on the card level, so you won't have to set styles for title and description text separately